app/views/admin/enterprises/form/_business_details.html.haml

```diff
@@ -31,3 +31,16 @@
     = f.label :invoice_text, t('.invoice_text')
   .omega.eight.columns
     = f.text_area :invoice_text, style: "width: 100%; height: 100px;"
+
+.row
+  .alpha.three.columns
+    = f.label :terms_and_conditions, t('.terms_and_conditions')
+
+  .omega.eight.columns
+    %a{ href: '{{ Enterprise.terms_and_conditions }}', ng: { if: 'Enterprise.terms_and_conditions' } }
```

Comment:
Why this is an `if` in javascript? Maybe you could check in Ruby/Haml and skip this work to browser when there are not terms and conditions uploaded